docker/install/ubuntu_install_python_package.sh

```diff
@@ -1,3 +1,3 @@
 # install libraries for python package on ubuntu
-pip2 install nose pylint numpy nose-timer cython decorator scipy tornado
-pip3 install nose pylint numpy nose-timer cython decorator scipy tornado typed_ast pytest mypy orderedset antlr4-python3-runtime
+pip2 install nose pylint numpy nose-timer cython decorator scipy tornado psutil
```

Comment:
seems the changes in docker is introduced un-intentionally?

Reply:
Fixed. I'll send this patch later.